lib/workers/repository/onboarding/branch/check.ts

```diff
@@ -52,8 +52,12 @@ function closedPrExists(config: RenovateConfig): Promise<Pr | null> {
 export async function isOnboarded(config: RenovateConfig): Promise<boolean> {
   logger.debug('isOnboarded()');
   const title = `Action required: Add a Renovate config`;
+
   // Repo is onboarded if global config is bypassing onboarding and does not require a
   // configuration file.
+  // Repo is not onboarded if the onboarding cache is present and
+  // the base branch has not been updated since last run
```

Comment:
Cache can also be missing on self-hosted renovate if not retained bewteen runs.

Reply:
It already says "if the onboarding cache is present"

Reply:
Ok, that sentense should be improved. the if should probably the first word, something like this:
```suggestion
  // If Repo is not onboarded, the onboarding cache is present and
  // the base branch has not been updated since last run
```

@HonkingGoose WDYT?

Reply:
```suggestion
  // The repo is considered "not onboarded" if:
  // - An onboarding cache is present, and
  // - The current default branch SHA matches the default SHA found in the cache
```

@viceice does this change make it more understandable?

Reply:
I like @rarkins suggestion, but it's not applied to the code yet.